python/src/debug.rs

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+use std::sync::Arc;
+
+use lance::{datatypes::Schema, Error};
+use lance_table::format::{DeletionFile, Fragment as LanceFragmentMetadata};
+use pyo3::{exceptions::PyIOError, prelude::*};
+
+use crate::{Dataset, FragmentMetadata, RT};
+
+/// Print the Lance schema of a dataset.
+///
+/// This can be used to view the field ids and types in the schema.
+#[pyfunction]
+pub fn print_schema(dataset: &PyAny) -> PyResult<()> {
+    let py = dataset.py();
+    let dataset = dataset.getattr("_ds")?.extract::<Py<Dataset>>()?;
+    let dataset_ref = &dataset.as_ref(py).borrow().ds;
+    let schema = dataset_ref.schema();
+    println!("{:#?}", schema);
```

Comment:
Why did you choose for these functions to actually `println` instead of returning a `String` to python?

Reply:
It felt natural at the time. I'm fine with making them return a string, but then maybe `print_` is the wrong prefix. What should I use instead?

Reply:
`format_fragment` is okay. I'll make it return a string and do that then.